src/cgame/rocket/rocketChatField.h

```diff
@@ -273,11 +273,18 @@ class RocketChatField : public Rml::Element, Rml::EventListener
 		Rml::GeometryUtilities::GenerateQuad( &vertices[0], &indices[0], Rml::Vector2f( 0, 0 ), cursor_size, GetProperty< Rml::Colourb >( "color" ) );
 	}
 
-	void MoveCursor( int amt )
+	void MoveCursorForward()
 	{
-		cursor_character_index += amt;
+		cursor_character_index += Q_UTF8_Width( text.c_str() + cursor_character_index );
```

Comment:
Why doesn't text[cursor_character_index] work?

Reply:
Q_UTF8_Width reads up to 4 `char`s.